From 6f384e12732a835a84a609f5e526d2cf9e808ea1 Mon Sep 17 00:00:00 2001
From: Tobias Zimmermann <tobias@the-resc.com>
Date: Wed, 20 Nov 2024 23:16:21 +0100
Subject: [PATCH] hwdb: Add quirk for Logitech MX Keys for Mac

The KEY_102ND and KEY_GRAVE keys are switched on the
Logitech MX Keys for Mac, so switch them back

(cherry picked from commit f70e5620b6ac319beac1d5ce26331bdc48d25daa)

Resolves: RHEL-49810
---
 hwdb.d/60-keyboard.hwdb | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 236a145dc5..6cc059123a 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -1438,6 +1438,11 @@ evdev:input:b0003v046DpC309*
  KEYBOARD_KEY_c01b6=images                              # My Pictures (F11)
  KEYBOARD_KEY_c01b7=audio                               # My Music (F12)
 
+# Logitech MX Keys for Mac
+evdev:input:b0003v046Dp4092*
+ KEYBOARD_KEY_70035=102nd                               # '<' key
+ KEYBOARD_KEY_70064=grave                               # '^' key
+
 ###########################################################
 # Maxdata
 ###########################################################
